src/yocto/getting-started.md

```diff
@@ -0,0 +1,26 @@
+# Getting started
```

Comment:
This is the yocto introduction, not getting started. getting started usually involves setup and/or first example